bucket/ghidra.json

```diff
@@ -0,0 +1,42 @@
+{
+    "version": "10.0.1",
+    "description": "Ghidra is a software reverse engineering (SRE) framework",
+    "homepage": "https://www.nsa.gov/ghidra",
+    "license": "Apache-2.0",
+    "extract_dir": "ghidra_10.0.1_PUBLIC",
+    "suggest": {
+        "JDK": "java/adoptopenjdk-lts-hotspot"
```

Comment:
Why JDK?? JRE is not enough?

Reply:
I continued the [installation instructions](https://github.com/NationalSecurityAgency/ghidra#install) in the document, which requests JDK.